wp-content/mu-plugins/pantheon.php

```diff
@@ -19,4 +19,16 @@
 	if ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) {
 		require_once 'pantheon/pantheon-login-form-mods.php';
 	}
+	if ( ! defined( 'FS_METHOD' ) ) {
+		/**
+		 * When this constant is not set, WordPress writes and then deletes a 
+		 * temporary file to determine if it has direct access to the filesystem,
```

Comment:
Would define the "it" here...

```suggestion
		 * temporary file to determine if the server has direct access to the filesystem,
```

Reply:
I think it == WordPress specifically